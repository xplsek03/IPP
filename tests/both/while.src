.IPPcode19
DEFVAR GF@counter
MOVE GF@counter string@
LABEL while
JUMPIFEQ end GF@counter string@aaaaa
WRITE string@counter\032obsahuje\032
WRITE GF@counter
WRITE string@\010
CONCAT GF@counter GF@counter string@a
JUMP while
LABEL end
