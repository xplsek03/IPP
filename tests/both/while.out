counter obsahuje 
counter obsahuje a
counter obsahuje aa
counter obsahuje aaa
counter obsahuje aaaa
